MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ListBox.xaml

```diff
@@ -124,7 +124,7 @@
                       Background="{TemplateBinding Foreground, Converter={StaticResource BrushRoundConverter}}"
                       Opacity="0" />
               <Border x:Name="SelectedBackgroundBorder"
-                      Background="{DynamicResource MaterialDesignDivider}"
+                      Background="{DynamicResource MaterialDesign.Brush.TextBox.HoverBackground}"
```

Comment:
Should this and the one below be a `MaterialDesign.Brush.ListBoxItem.xxx` variant instead? It does not appear to be a nested `TextBox` or similar.

Reply:
Fixed